src/universal/components/LinkNewTab/LinkNewTab.js

```diff
@@ -0,0 +1,28 @@
+import React, {PropTypes} from 'react';
+
+const LinkNewTab = (props) => {
+  const { children, href, title } = props;
+  let url = href;
+  // Add http protocol to url if no protocol was provided
+  const urlProtocolRegex = new RegExp(/http:\/\/|https:\/\/|mailto:|ftp:\/\//);
```

Comment:
That's what I thought. I wanted to ask first, however.

May I suggest that you change this in one of two following ways?

1. You change the regex to be expanded to all protocol types (e.g. /^[a-zA-Z]+:/), or,
2. You remove the check altogether, and we enforce properly formatted URLs.

Reason being, what if I want to link to a local file? What if I want to link to something on an intranet smb share?

I'm rather pragmatic about internet standards :)

I like the spirit behind the effort, though!